### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by ccw12345** [wzjialang/MFPNet#3]
In your paper, you set the learning rate was decayed with a cosine annealing schedule。I want to know the specific value of the parameter，T_max and eta_min. Thank you!

**Comment by wzjialang** [comment]
The hyper-parameters of lr scheduler can be seen below:

scheduler = torch.optim.lr_scheduler.CosineAnnealingWarmRestarts(optimizer, T_0=10, T_mult=2, eta_min=0, last_epoch=-1)

**Comment by ccw12345** [comment]
thanks
